integration/client/go.mod

```diff
@@ -35,11 +35,11 @@ require (
 	github.com/godbus/dbus/v5 v5.0.6 // indirect
 	github.com/gogo/googleapis v1.4.0 // indirect
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
-	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/golang/protobuf v1.5.3 // indirect
 	github.com/google/go-cmp v0.5.9 // indirect
 	github.com/google/uuid v1.3.0 // indirect
 	github.com/imdario/mergo v0.3.12 // indirect
-	github.com/klauspost/compress v1.11.13 // indirect
+	github.com/klauspost/compress v1.15.9 // indirect
```

Comment:
Interesting one; I was actually looking _why_ it updated, but I'm guessing that integration/client uses some dependencies that are not used in the main module; and because of that it forces it to be updated;

```
github.com/klauspost/compress  has different values in the go.mod files require section:  v1.11.13 in root go.mod  v1.15.9 in integration/client/go.mod
```


Updating seems like a sane thing to do, so let me add a commit to update the version

Reply:
Added https://github.com/containerd/containerd/pull/7325